src/main/resources/reference.conf

```diff
@@ -803,22 +814,30 @@ resourceTypes = {
       modify = {
         description = "modify attributes of the cluster"
       }
+      set_parent = {
+        description = "set parent of notebook cluster"
+      }
     }
     ownerRoleName = "creator"
     roles = {
       creator = {
         roleActions = ["status", "connect", "delete", "read_policies", "stop_start", "modify"]
       }
+      manager = {
+        roleActions = ["status", "delete", "read_policies"]
+      }
     }
     reuseIds = false
+    authDomainConstrainable = true
   }
   persistent-disk = {
     actionPatterns = {
       read = {
-        description = "read metadata and contents of persistent disk"
+        description = "read metadata of persistent disk"
```

Comment:
This is just used for listing disks; `attach` is needed to actually view the contents of a disk